### ISSUE TITLE
AS2 worked once, long ago...

### ISSUE THREAD

**Issue by AkkiWolf** [jadedcat/AS2Configs#96]
A bit after initial release, AS2 worked, once I tried to update (via CV) it never worked again, even when I rolled back to the version that worked. The below is what I've gotten everytime I've tried to run AS2 any fixes would be welcome since I don't really understand what the program is trying to do.
http://pastebin.com/sMy2r1Tr (minecraft log from MultiMC)
http://pastebin.com/37wsZXpm (FML-client-Latest)
AS2 Version 1.1.13
Launching from the launcher

**Comment by MarshMilo100** [comment]
Try completely uninstalling ag skies 2 and reinstalling it with cv.

**Comment by pjfranke** [comment]
You are running this via CurseVoice, or MultiMC?

If you just "install" the pack to create a new instance, and try playing that, does it work?  What about a different pack...does any other Minecraft 1.7.10 pack work via CurseVoice?

**Comment by AkkiWolf** [comment]
The above was with CV, as for other packs, I'm having difficulty updating FTB Inf, from CV as well, it claims theres dup mods in the file but there aren't near as I can see. I tried Daybreaker from CV as well, and there was issues getting it to run...tbh I'm stumped, unless it's Kaspersky or some other program interfering.

**Comment by pjfranke** [comment]
Well you can always download the "alternative" zip and play it that way.  Or, hmm..I have another idea, I'll need to test it and follow up.

**Comment by AkkiWolf** [comment]
PJ, this is Redwolf1313 from the other board, on how to install AS2. I've tried everything short of dragging out a how to mod the vanilla launcher...which may or may not work. AFIK both my FTB and AT launcher run as intended (I've only tested FTB w/ an Inf update) so the CV and MultiMC really have me stumped since just about everywhere I read people are praising one for ease of use, and the utillity of the other (make your own judgement there). I may have to test this on my wife's machine, but I'm actually beginning to think it's something to do with my setup.

**Comment by enarkay** [comment]
I know this may sound really lame, but have you tried to completely erase & remove the CV launcher and its folder? I've kind of experienced the similar problem as you did, although in my case this happened after I added an extra mod I needed to AS2. Seeing that the modpack didn't load, I figured the extra mod wouldn't work with AS2 so I simply uninstalled AS2 and tried again but it still didn't work. So I ended up completely removing the CV launcher and its folders, reinstalled CV, and it worked from then on. But yeah, I guess the CV does that sometimes...
